src/cjson/lua_cjson.c

```diff
@@ -1392,12 +1404,39 @@ static int json_decode(lua_State *l)
 {
     json_parse_t json;
     json_token_t token;
+    json_options_t options = { .luanil_object = false, .luanil_array = false };
+
+
     size_t json_len;
 
-    luaL_argcheck(l, lua_gettop(l) == 1, 1, "expected 1 argument");
+    switch (lua_gettop(l)) {
+    case 1:
+        break;;
```

Comment:
double `;;`  (also below)

Reply:
done (I just yanked the switch/case from above)